### PR TITLE
Age Calculator #91

### PR DESCRIPTION
**Pull request by Abhay04ch**
# Pull Request for Play With Python 💡

## Issue Title : Age Calculator

- **Info about the related issue (Aim of the project)** : This project aims to create an age calculator app.
- **Name:** Abhay Chauhan
- **GitHub ID:** https://github.com/Abhay04ch
- **Email ID:** chauhan04abhay@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in.)** CODEPEAK 22

Closes: #91 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested

#### I have run the code on my device and tested each app feature.

Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Age Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+6. Used different fonts, and colour mode to make it attractive for GUI.
+
+### LIBRARIES NEEDED
+
+1. Thinkter: Provides built-in functions to create GUI.
+2. Messagebox: It comes under thinkter library and it is used to create buttons.
+
+### DEMONSTRATION
+
+1. Image1 - First image shows the app preview.
```
Comment (abhisheks008) [inline Age Calculator/README.md:27]: Insert your images by hyperlinking them,
Syntax: `Image Name`

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Age Calculator/README.md (hunk 1 above)
Comment (Abhay04ch) [reply Age Calculator/README.md:27]: okay

**Comment by Abhay04ch** [comment]
@abhisheks008 I have made the changes. Please do review them.

**Review by abhisheks008 — APPROVED** [review]
Now it's looks good. Your PR is approved and ready to be merged.
@Abhay04ch

**Pull request merged**